src/components/recruting/_04_interview_notification/_01/PassList.tsx

```
@@ -0,0 +1,74 @@
+  return (
+    <div className="w-max-[440px] pr-[10px]">
+      <section className="flex items-center gap-[18px]">
+        <div className="flex items-center">
+          <div className="text-[#525252] text-[13.3px] mr-[5px] p-[6px] rounded-[5px] bg-white-100 border border-gray-200">
+            전체 합격자
+          </div>
+          <p className="text-[#6F7278] text-[11px]">{"60"}</p>
+        </div>
+        {groups.map((group) => (
+          <div className="flex items-center">
+            <div className="text-[#525252] text-[13.3px] mr-[5px] p-[6px] rounded-[5px] bg-white-100 border border-gray-200">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)